### PR TITLE
ex23_ch4.py

### PR DESCRIPTION
**Pull request by amymhaddad**
Create a program that walks the user through troubleshoot- ing issues with a car. See attached image.

### CODE REVIEW

**Comment by amymhaddad** [comment]
screen shot 2018-07-21 at 8 17 18 am

File: ex23_ch4.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    else: 
+        print("Replace the cables and try it again.")
+
+#step3: write second conditional based on input from user
+elif turn_key == "no":
+    click_sound = input("Does the car make a clicking noise? Enter \"yes\" or \"no\": ")
+    normalized_click_sound = click_sound.lower()
+    if click_sound == "yes":
+        print("Replace the battery.")
+    elif click_sound == "no":
```
Comment (amymhaddad) [inline ex23_ch4.py]: I intentionally use "elif" here. Is this the best option? I wouldn't use "else" here, correct?

Comment (paulghaddad) [reply ex23_ch4.py]: I agree here. The only thing to note is what happens if they don't enter `yes` or `no`. However, I think it's fine for the purposes of this exercise.

**Comment by amymhaddad** [comment]
@paulghaddad Is there a change to make on this one?

**Comment by paulghaddad** [comment]
@amymhaddad Nope, it's good. Already merged.

**Pull request merged**